build.sh

```diff
@@ -133,16 +134,16 @@ install_all_deps()
 	if [ ${platform} -eq ${Ubuntu_Platform} ];then
 		install_ubuntu_deps
 		if [ $enable_guomi -eq 0 ];then
-			execute_cmd "sudo cp fisco-solc-ubuntu  /usr/bin/fisco-solc && sudo chmod +x /usr/bin/fisco-solc"
+			execute_cmd "sudo curl -Lo /usr/bin/fisco-solc ${solc_link_prefix}-ubuntu && sudo chmod +x /usr/bin/fisco-solc"
```

Comment:
I found that you have a pr to update these into master, why don't you merge from master after that pr ?